java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -649,6 +649,11 @@ private static native long conditionalLeftAntiJoinRowCount(long leftTable, long
 
   private static native long[] gather(long tableHandle, long gatherView, boolean checkBounds);
 
+  private static native long[] scatter(long srcTableHandle, long scatterView,
+                                            long targetTableHandle, boolean checkBounds);
+  private static native long[] scatter(long[] srcScalarHandles, long scatterView,
+                                             long targetTableHandle, boolean checkBounds);
```

Comment:
I don't think we want to overload on this native method.  It's not very friendly on the JNI end, as you have to name your native C method based on the mangled argument signature (which is not done below, so this is disconnected from the JNI C++ code and will fail with a symbol lookup error).  Given these are private interfaces, I see no advantage to overloading on the name but definitely see some downsides.

Reply:
Sorry, I implemented them with different names at the beginning but then changed them. Now I reversed back.